src/main/java/Duke.java

```diff
@@ -1,10 +1,189 @@
+
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.stream.*;
+
+
 public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void talk() {
```

Comment:
Perhaps creating new classes (Storage/UI) would help to abstract this huge line of code?